source/features/index.tsx

```diff
@@ -59,11 +59,15 @@ const logError = (url: string, error: unknown): void => {
 	// Don't change this to `throw Error` because Firefox doesn't show extensions' errors in the console
 	console.group('❌', id, version, pageDetect.isEnterprise() ? 'GHE →' : '→', error);
 
+	const searchIssueUrl = new URL('https://github.com/refined-github/refined-github/issues');
+	searchIssueUrl.searchParams.set('q', `is:issue is:open sort:updated-desc "${message}"`);
 	console.group('Search issue');
-	console.log(`https://github.com/refined-github/refined-github/issues?q=is%3Aissue+${encodeURIComponent(message)}`);
+	console.log(searchIssueUrl.href);
 	console.groupEnd();
 
-	const newIssueUrl = new URL('https://github.com/refined-github/refined-github/issues/new?labels=bug&template=1_bug_report.md');
+	const newIssueUrl = new URL('https://github.com/refined-github/refined-github/issues/new');
+	newIssueUrl.searchParams.set('labels', 'bug');
```

Comment:
This should be handled by the form already

Reply:
Nope, try it yourself: https://github.com/refined-github/refined-github/issues/new?template=1_bug_report.yml

![image](https://user-images.githubusercontent.com/44045911/143929303-ab6ad541-e67e-4c87-94e1-5893963a0890.png)

Reply:
Try it yourself 😉 

Non-maintainers cannot specify labels, only the template can. What those parameters do is change the immediate view, for example the title isn't automatically set even if the template includes it.

You can see that the labels are missing from the sidebar, but they are added anyway.

Original link: https://github.com/typescript-eslint/typescript-eslint/issues/new?assignees=&labels=enhancement%3A+new+base+rule+extension%2Cpackage%3A+eslint-plugin%2Ctriage&template=eslint-plugin-typescript.yml&title=Rule+proposal%3A+

Link I used for this video: https://github.com/typescript-eslint/typescript-eslint/issues/new?template=eslint-plugin-typescript.yml

https://user-images.githubusercontent.com/1402241/144068558-84e17e02-ef73-4c9a-8e95-f38ca619f784.mov

Reply:
Damn GitHub 😂 https://github.com/refined-github/refined-github/issues/5135

Let's keep the `labels` parameter

Reply:
_[Confused Pikachu]_

Reply:
Good as-is